plugins/frontend/backstage-plugin-shortcut/src/api/ShortcutClient.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ShortcutApi } from './ShortcutApi';
+import { Story, User } from './types';
+import { DiscoveryApi } from '@backstage/core-plugin-api';
+
+const DEFAULT_PROXY_PATH = '/shortcut/api';
+
+export type Options = {
+  discoveryApi: DiscoveryApi;
+  proxyPath?: string;
+};
+
+export class ShortcutClient implements ShortcutApi {
```

Comment:
I think this client would be worth to write some tests for the public functions. It seems there are not much happening but I'd check if it throws an error and fetches from the correct path.